src/components/LoanCards/Buttons/LendButton.vue

```diff
@@ -71,6 +71,8 @@ export default {
 							);
 							Sentry.captureMessage(`Add to Basket: ${error.message}`);
 							if (error?.extensions?.code === 'shop.invalidBasketId') {
+								// eslint-disable-next-line max-len
+								this.$showTipMsg('There was a problem adding the loan to your basket, refreshing the page to try again.', 'error');
```

Comment:
With this added, I think we'll need to use an `else` block below to prevent 2 messages from showing.

Reply:
`return handleInvalidBasket` is preventing second tip message to show

Reply:
Ahh ok, Thanks!